metastats-domain/src/main/groovy/life/qbic/metastats/MetaStatsLogManager.groovy

```diff
@@ -0,0 +1,36 @@
+package life.qbic.metastats
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since: <versiontag>
+ * @author: Jennifer Bödker
+ *
+ */
+interface MetaStatsLogManager {
+    /**
+     * Logs a debug message
+     * @param message which is logged
```

Comment:
I'd suggest making the docs for each method in this class distinguishable from each other
```suggestion
     * @param  message containing debugging information
```